code/modules/mob/living/say.dm

```diff
@@ -88,9 +90,12 @@ proc/get_radio_key_from_channel(var/channel)
 	if(GREY in mutations)
 		message = "<span class='grey'>[message]</span>"
 
-	else if(COMIC in mutations)
+	if(COMIC in mutations)

```

Comment:
@Fox-McCloud the point is making sure that only one class is applied, to avoid inheritance issues

sure right now maybe comic overrides what the grey mutation does, but what about when it doesn't?